workerapi/client/client.go

```diff
@@ -0,0 +1,115 @@
+package client
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/apache/thrift/lib/go/thrift"
+	"github.com/scootdev/scoot/runner"
+	"github.com/scootdev/scoot/workerapi"
+	"github.com/scootdev/scoot/workerapi/gen-go/worker"
+)
+
+type Client interface {
```

Comment:
This, and the fact that you have to panic in one, is code smell.

Fortunately, it's easy to avoid. Just have NewCliClient return a *CliClient.